tests/unit/aiplatform/test_datasets.py

```diff
@@ -147,6 +148,30 @@
 
 _TEST_LABELS = {"my_key": "my_value"}
 
+# create_from_dataframe
```

Comment:
Preference for an integration test as well.

Reply:
Added 2 integration tests: one with the `bq_schema` param and one without.